adoc/scripts/spec_tools/conventions.py

```diff
@@ -0,0 +1,454 @@
+#!/usr/bin/python3 -i
+#
+# Copyright 2013-2023 The Khronos Group Inc.
```

Comment:
I assume that @TApplencourt started by copying the "conventions.py" file from the Vulkan repo, which has this copyright date range.  Since this file is a derived work, I think it's correct to keep the original date range.